### PR TITLE
Prototype announcement

### PR DESCRIPTION
**Pull request by aleone2**
Updated main page to include prototype announcements and links

### CODE REVIEW

**Comment by aleone2** [comment]
[Announce Prototype release on Website](https://trello.com/c/SGWe16EG/78-announce-prototype-release-on-website)

**Comment by aleone2** [comment]
[Announce Prototype release on Website](https://trello.com/c/SGWe16EG/78-announce-prototype-release-on-website)

**Comment by chase-saca** [comment]
Screen Shot 2022-03-02 at 9 37 50 AM

I don't think we need all four of these. Two of them are on the game platform release and two are on the trivalry release so it seems repetitive?

File: src/containers/BlogTemplate.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ import FeaturedPost from './FeaturedPost';
 import BubblesBackground from "../images/bubbles.png";
 import GameConsoles from '../images/gameconsoles.png';
 import GameGraphic from '../images/gamegraphic.png';
+//import Loading from '../images/loading.png';
```
Comment (rpmccarter) [inline src/containers/BlogTemplate.js:8]: can delete this

File: src/containers/BlogTemplate.js [hunk 2]
```diff
@@ -16,21 +18,46 @@ const mainFeaturedPost = {
...
         date: 'Date TBD',
         description:
             'This is the launch of PartyFish\'s first game, Trivalry, a fun and exciting mix of trivia and survivor.',
-        image: GameGraphic,
+        image: Loading,
         imageLabel: 'Image Text',
+        link: '#',
+        announcement: 'Stay Tunned...',
     },
+*/
```
Comment (rpmccarter) [inline src/containers/BlogTemplate.js:60]: probably safe to delete this as well

File: src/containers/Documentation.js [hunk 3]
```diff
@@ -6,6 +6,7 @@ import Typography from '@mui/material/Typography';
 import Box from '@mui/material/Box';
 import Paper from '@mui/material/Paper';
 import Link from "@mui/material/Link";
+import { LinkContainer } from 'react-router-bootstrap';
 //import CssIcon from '@mui/icons-material/Css';
 //import HtmlIcon from '@mui/icons-material/Html';
 //import JavascriptIcon from '@mui/icons-material/Javascript';
```
Comment (rpmccarter) [inline src/containers/Documentation.js:12]: delete

File: src/containers/FeaturedPost.js [hunk 4]
```diff
@@ -6,27 +6,37 @@ import Card from '@mui/material/Card';
...
               {post.description}
             </Typography>
-            <Typography variant="subtitle1" color="primary">
-              Stay tuned...
-            </Typography>
+            <Box sx={{border: '1px dashed #d6a304'}}>
+              <Typography variant="subtitle1" color="primary">
+                {post.announcement}
+              </Typography>
+            </Box>
```
Comment (rpmccarter) [inline src/containers/FeaturedPost.js:39]: super clean, love it

File: src/containers/Installation.js [hunk 5]
```diff
@@ -1,29 +1,156 @@
...
+  },
+  {
+    text: "4-8 Super Fun Friends",
+    icon: <GroupAddIcon sx={{color:"#023E8A"}}/>
+  },
+  {
+    text: "To be prepared to have FUN",
+    icon: <CelebrationIcon sx={{color:"#023E8A"}}/>
+  },
+]
```
Comment (rpmccarter) [inline src/containers/Installation.js:156]: love this

File: src/containers/FeaturedPost.js (hunk 4 above)
Comment (aleone2) [reply src/containers/FeaturedPost.js:39]: blog template!

**Review by rpmccarter — APPROVED** [review]
Looks great!

**Comment by aleone2** [comment]
> 

taken care of!

**Pull request merged**